pulsar-functions/utils/src/main/java/org/apache/pulsar/functions/utils/FunctionConfigUtils.java

```diff
@@ -35,8 +35,6 @@
 import java.lang.reflect.Type;
 import java.net.MalformedURLException;
 import java.util.*;
```

Comment:
@vzhikserg Thanks for the fix. Would you please also help remove this `*` use case? we should avoid `*`

Reply:
Sure, it is fixed